src/System.Net.Security/tests/FunctionalTests/NegotiateStreamInvalidOperationTest.cs

```diff
@@ -0,0 +1,234 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Net.Test.Common;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)] // NegotiateStream only supports client-side functionality on Unix
+    public class NegotiateStreamInvalidOperationTest
+    {
+        private static readonly byte[] s_sampleMsg = Encoding.UTF8.GetBytes("Sample Test Message");
+
+        [Fact]
+        public async Task NegotiateStream_StreamContractTest_Success()
+        {
+            VirtualNetwork network = new VirtualNetwork();
```

Comment:
nit: could be
```c#
var network = new VirtualNetwork();
```

comment applies to other occurrences in PR.

Reply:
Will fix.